.github/workflows/build-open-fleet.yml

```diff
@@ -6,54 +6,15 @@ on:
   workflow_dispatch:
 
 env:
-  balena-cli: v15.1.1
-  raspberrypi3-64: 2.113.18
-  raspberrypi4-64: 2.113.18
-  raspberrypicm4-ioboard: 2.112.12
-  rockpi-4b-rk3399: 2.108.25+rev1
-  cloud: nebra-cloud
+  : v15.1.1
```

Comment:
This is a typo I guess? Maybe needs to be balena-cli ?

Reply:
Yes, that's a typo - I must have deleted that one by mistake right before committing or so.